modules/upgrade-central-cluster-crds.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-roxctl.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="upgrade-central-cluster-crds{context}"]
+= Upgrading `SecurityPolicy` CRD
+
+[role="_abstract"]
+You can update the `SecurityPolicy` custom resource definitions (CRD) to the latest version by generating the new CRD and applying it to the cluster.
+
+.Procedure
+
+* Use `roxctl` to generate a new set of resources
```

Comment:
```suggestion
. Use `roxctl` to generate a new set of resources by entering the following command:
```
It looks like these should be numbered steps - to achieve that, use `.` instead of `*`.

The part about "running the following command" needs to be added because of localization requirements.